### PR TITLE
Bookstore styling

### PR DESCRIPTION
**Pull request by AlexisJW**
This pull request concerns the list of tasks according to the [assignment](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/styling.md).

Project requirements

- Add any missing JSX elements to my project (without adding extra functionality)
- Final design should match Zeplin's design (this includes all elements + the same styling)
- Use the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/styles/index.scss [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    .horizontal-divider {
+      width: 76rem;
+      height: 0.115rem;
+      border: solid 2px var(--white);
+      margin: 3.75rem 5.795rem 0.125rem 5.795rem;
+      background-color: #c4c4c4;
+    }
+  }
+}
```
Comment (Owusu-Desmond) [inline src/styles/index.scss]: - [ ] Please as you can see from the video below, Your project is having horizontal overflow which makes your `Add Book` button and `user-icon` not show on the page unless the page is scrolled horizontally. Please make sure you remove the horizontal overflow and make the design looks exactly like the Zeplin template :relaxed:

[Screencast from 03-04-2023 03:27:03 AM.webm](https://user-images.githubusercontent.com/92303891/222873568-5fd4dc60-b7e4-410a-a3ec-e85cb14ad6dd.webm)

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @AlexisJW , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/index.scss (hunk 1 above)
Comment (AlexisJW) [reply src/styles/index.scss]: Thank you Mr. @Owusu-Desmond ! I'm working on it!

File: src/styles/index.scss [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    width: 87.5rem;
+    height: auto;
+    margin: 0 2.5rem 2.063rem 0;
+    padding: 0 0 4.625rem;
+    border-radius: 4px;
+    box-shadow: 0 0 90px 0 var(--black-5);
+    background-color: var(--z);
+
+    .horizontal-divider {
+      width: 76rem;
```
Comment (DuaneDave) [inline src/styles/index.scss]: As previously requested, the horizontal scroll issue has not been resolved, kindly resolve this issue as it makes the UI quite distorted and provides an overall bad UX.

See the video below.



https://user-images.githubusercontent.com/84930355/222921587-604664da-d507-4731-9a73-987144e65269.mp4

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @AlexisJW 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights
✔️ Descriptive PR summary
✔️ Good implementation of react hooks
✔️ Meaningful commit history

 ## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @AlexisJW, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>

## Good Highlights✔️

- Requested change has been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**